compiler/rustc_passes/src/check_attr.rs

```diff
@@ -705,7 +705,7 @@ impl CheckAttrVisitor<'tcx> {
         let mut is_valid = true;
 
         if let Some(list) = attr.meta().and_then(|mi| mi.meta_item_list().map(|l| l.to_vec())) {
-            for meta in list {
+            for meta in &list {
```

Comment:
What is the reason behind this change?

Reply:
It avoids a borrow error below, where I call `list.len()`.